Homework/Homework/Program.cs

```diff
@@ -180,4 +183,20 @@ public static int GetElementCount(int[] arr, int itemToCount)
         }
         return count;
     }
+    public static double MaxIndex(double[] array)
+    {
+        double maxNumber = 0;
+        int indexOfNumb = 0;
+        for (double i = 0; array.Length > i; i++)
+        {
+            if (maxNumber < array[(int)i])
+            {
+                maxNumber = array[(int)i];
+                indexOfNumb = (int)i;
+            }
```

Comment:
в принципе решение верное, только тут тебя не туда понесло, смотри как бы сделал я
```suggestion
            if (maxNumber < array[i])
            {
                maxNumber = array[i];
                indexOfNumb = i;
            }
```